.buffy/templates/reviewer_checklist.md

```diff
@@ -31,8 +31,8 @@
 
 ### Paper format
 
-- [ ] **Authors:** Does the `paper.tex` file include a list of authors with their affiliations?
-- [ ] **A statement of need:** Does the paper have a section titled 'Statement of need' that clearly states what problems the software is designed to solve, who the target audience is, and its relation to other work?
+- [ ] **Authors:** Does the `paper.pdf` file include a list of authors with their affiliations?
+- [ ] **A statement of need:** Does the paper clearly state what problems it is designed to solve, who the target audience is, and its relation to other work?
```

Comment:
Maybe we miss that the software should address some challenges or problem. Also, we could try phrasing things to highlight it should "describe the implications and importance of the problem to the wider community" as well. I currently just struggle to find good wording...

Reply:
a bit plain, but what about

```suggestion
- [ ] **A statement of need:** Does the paper clearly state what problems it is designed to solve, how that is relevant to the wider community, who the target audience is, and its relation to other work?
``

Reply:
or maybe
```suggestion
- [ ] **A statement of need**: Does the paper clearly explain the problem it addresses, its importance to the broader community, the intended audience, and how it connects to existing work in the field?
```

Reply:
I like the second option